### PR TITLE
PLATUI: Update examples

### PR DESCRIPTION
**Pull request by matthewmascord**

### CODE REVIEW

**Comment by matthewmascord** [comment]
retest this please

File: build.sbt [hunk 1]
```diff
@@ -40,7 +43,6 @@ lazy val root = Project(libName, file("."))
     },
     generateExamplesManifest := {
       println("==========")
-      val _                      = generateExamples.value
```
Comment (JoPintoPaul) [inline build.sbt:43]: I see that this value was never used, but was this not the command to generate the examples? Or is it done aas a dependency of the `generateExamplesManifest` task?

File: project/LibDependencies.scala [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+        "com.typesafe.play" %% "filters-helpers" % PlayVersion.current,
         "org.webjars.npm"    % "govuk-frontend"  % govukFrontendVersion,
         "org.joda"           % "joda-convert"    % "2.0.2",
         "com.lihaoyi"       %% "fastparse"       % "2.1.2"
       )
 
       val test = Seq(
+        "com.vladsch.flexmark"          % "flexmark-all"     % "0.35.10",
         "org.scalatest"                %% "scalatest"        % "3.0.8",
         "org.pegdown"                   % "pegdown"          % "1.6.0",
```
Comment (JoPintoPaul) [inline project/LibDependencies.scala:26]: I could be wrong, but might be worth checking if `pegdown` can be removed.

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
+  sys.env.get("PLAY_VERSION") match {
+    case Some("2.8") => "com.typesafe.play" % "sbt-plugin" % "2.8.7"
+    case Some("2.7") => "com.typesafe.play" % "sbt-plugin" % "2.7.9"
+    case _           => "com.typesafe.play" % "sbt-plugin" % "2.6.25"
+  }
+)
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.13.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
```
Comment (JoPintoPaul) [inline project/plugins.sbt:19]: If `sbt-auto-build` goes to 3.0.0, you can remove `sbt-git-versioning`.

File: src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/changeList.scala.html [hunk 4]
```diff
@@ -46,13 +46,13 @@ <h1 class="govuk-heading-xl">Change Elizabeth Keen’s details</h1>
...
         ))
-      ),
+      ), 
       SummaryListRow(
         key = Key(
           content = Text("Does Elizabeth Kean have a National Insurance number?")
         ),
         value = Value(
-          content = HtmlContent("Yes")
+          content = HtmlContent(Yes)
```
Comment (JoPintoPaul) [inline src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/changeList.scala.html:55]: Does this work? I thought the `Yes` needed to the wrapped as a string `"Yes"`

File: src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/checkAnswers.scala.html [hunk 5]
```diff
@@ -47,13 +47,13 @@ <h1 class="govuk-heading-l">Check your answers</h1>
...
         ))
-      ),
+      ), 
       SummaryListRow(
         key = Key(
           content = Text("Does Elizabeth Kean have a National Insurance number?")
         ),
         value = Value(
-          content = HtmlContent("Yes")
+          content = HtmlContent(Yes)
```
Comment (JoPintoPaul) [inline src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/checkAnswers.scala.html:56]: See previous comment.

File: src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/hmrcbanner/example.scala.html [hunk 6]
```diff
@@ -5,4 +5,4 @@
 @()
 
 
-@hmrcBanner(Banner())
+@hmrcBanner()
```
Comment (JoPintoPaul) [inline src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/hmrcbanner/example.scala.html:8]: Does this definitely compile?

File: src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/welshlanguagetoggle/example.scala.html [hunk 7]
```diff
@@ -7,6 +7,14 @@
 
 @hmrcLanguageSelect(LanguageSelect(
     language = En,
-    En -> "?en",
-    Cy -> "?cy"
-))
+    languageLinks = ArrayBuffer(
```
Comment (JoPintoPaul) [inline src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/welshlanguagetoggle/example.scala.html:10]: I think this change needs rolling back.

File: src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/welshlanguagetoggle/exampleWelsh.scala.html [hunk 8]
```diff
@@ -7,6 +7,14 @@
 
 @hmrcLanguageSelect(LanguageSelect(
     language = Cy,
-    En -> "?en",
-    Cy -> "?cy"
-))
+    languageLinks = ArrayBuffer(
```
Comment (JoPintoPaul) [inline src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/welshlanguagetoggle/exampleWelsh.scala.html:10]: Think this should be rolled back.

File: src/test/play-28/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/changeList.scala.html [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+            )
+          )
+        ))
+      ), 
+      SummaryListRow(
+        key = Key(
+          content = Text("Does Elizabeth Kean have a National Insurance number?")
+        ),
+        value = Value(
+          content = HtmlContent(Yes)
```
Comment (JoPintoPaul) [inline src/test/play-28/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/changeList.scala.html:55]: See previous comment.

File: src/test/play-28/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/checkAnswers.scala.html [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+            )
+          )
+        ))
+      ), 
+      SummaryListRow(
+        key = Key(
+          content = Text("Does Elizabeth Kean have a National Insurance number?")
+        ),
+        value = Value(
+          content = HtmlContent(Yes)
```
Comment (JoPintoPaul) [inline src/test/play-28/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/checkAnswers.scala.html:56]: See previous comment.

File: build.sbt (hunk 1 above)
Comment (matthewmascord) [reply build.sbt:43]: Nope, this just makes it re-generate the examples whenever generating the manifest, which was overwriting some manual changes I had been making.

File: src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/changeList.scala.html (hunk 4 above)
Comment (matthewmascord) [reply src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/changeList.scala.html:55]: Yes, it appears the play-27 and play-28 directories are not actually compiled, which is a bit worrying.

File: project/LibDependencies.scala (hunk 2 above)
Comment (matthewmascord) [reply project/LibDependencies.scala]: Removed for 2.8, but still needed for earlier versions.

**Review by JoPintoPaul — APPROVED** [review]

**Pull request merged**